website/docs/r/s3_bucket.html.markdown

```diff
@@ -458,6 +479,13 @@ The `apply_server_side_encryption_by_default` object supports the following:
 * `sse_algorithm` - (required) The server-side encryption algorithm to use. Valid values are `AES256` and `aws:kms`
 * `kms_master_key_id` - (optional) The AWS KMS master key ID used for the SSE-KMS encryption. This can only be used when you set the value of `sse_algorithm` as `aws:kms`. The default `aws/s3` AWS KMS master key is used if this element is absent while the `sse_algorithm` is `aws:kms`.
 
+The `grant` object supports the following:
+
+* `id` - (optional) Canonical user id to grant for. Used only when `type` is `CanonicalUser`.  
+* `type` - (required) - Type of grantee to apply for. Valid values are `CanonicalUser` and `Group`.
```

Comment:
Missing `AmazonCustomerByEmail` in the documentation"

```suggestion
* `type` - (required) - Type of grantee to apply for. Valid values are `AmazonCustomerByEmail`, `CanonicalUser`, and `Group`.
```

Reply:
Answered below. No support for `AmazonCustomerByEmail`

Reply:
done